internal/cli/service.go

```diff
@@ -146,6 +146,28 @@ func (s Service) InitiateRun(cfg InitiateRunConfig) (*api.InitiateRunResult, err
 	if err != nil {
 		return nil, errors.Wrap(err, "unable to read provided files")
 	}
+	if len(taskDefinitions) != 1 {
+		return nil, fmt.Errorf("Expected exactly 1 task definition, got %d", len(taskDefinitions))
```

Comment:
I don't think this nomenclature is right given how we talk publicly. Is this a run definition file?

Reply:
You're right, I'll update the text.